tests/agent_server/test_facts.py

```diff
@@ -226,8 +226,9 @@ async def test_get_facts_extended(server, client, agent, clienthelper, resource_
     ]
 
     resource_states = {
-        "test::Fact[agent1,key=key4],v=%d" % version: const.ResourceState.undefined,
-        "test::Fact[agent1,key=key5],v=%d" % version: const.ResourceState.undefined,
```

Comment:
I think including the resource versions in here was a bug. The `resource_state` arg of `put_version()` is typed as `dict[ResourceIdStr, ...]` And this check was failed because of it: https://github.com/inmanta/inmanta-core/blob/48a780475ae561c988deefc0ba667b919ca98793/src/inmanta/server/services/orchestrationservice.py#L573

Reply:
I think you're right.